.travis.yml

```diff
@@ -67,22 +67,20 @@ addons:
       version: "0.2.0-SNAPSHOT"
       description: "Scalable, distributed graph database"
     notification_email: janusgraph-ci@googlegroups.com
-    # For more info on `travis_wait`, see the docs:
-    # https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
     build_command: "mvn clean package -B -DskipTests=true"
     branch_pattern: coverity_scan
 
 install:
-  # Build and install current module and dependencies. Handle output timeouts and retry on error.
-  - travis_retry travis_wait mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -pl janusgraph-${MODULE} -am
+  - if [ "${COVERITY_SCAN_BRANCH}" != 1 ]; then
+      # Build and install current module and dependencies. Handle output timeouts and retry on error.
+      travis_retry travis_wait mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -pl janusgraph-${MODULE} -am;
```

Comment:
Can you split this mvn install command into multiple lines and comment on each command line argument?